### PR TITLE
fix: backend expects secret header

### PR DESCRIPTION
**Pull request by fredmaggiowski**

### CODE REVIEW

**Comment by fredmaggiowski** [comment]
Sorry @davidebianchi there was a problem with previous MR

**Comment by davidebianchi** [comment]
We could use `client_key` as header instead of `secret`. It could be better?

File: sdk/client.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ func New(opts Options) (*MiaClient, error) {
 		BaseURL: opts.APIBaseURL,
 		Headers: map[string]string{
 			"cookie": opts.APICookie,
-			"apiKey": opts.APIKey,
+			"secret": opts.APIKey,
```
Comment (fredmaggiowski) [inline sdk/client.go:47]:
```suggestion
			"client-key": opts.APIKey,
```

File: sdk/client_test.go [hunk 2]
```diff
@@ -46,7 +46,7 @@ func TestNew(t *testing.T) {
 		expectedJSONClient, _ := jsonclient.New(jsonclient.Options{
 			BaseURL: opts.APIBaseURL,
 			Headers: map[string]string{
-				"apiKey": opts.APIKey,
+				"secret": opts.APIKey,
```
Comment (fredmaggiowski) [inline sdk/client_test.go:49]:
```suggestion
				"client-key": opts.APIKey,
```

**Pull request merged**